### PR TITLE
Make rude gifs a link rather than in-lining them

### PR DESCRIPTION
**Pull request by meadsteve**

### CODE REVIEW

File: extensions/gif-me.js [hunk 1]
```diff
@@ -1,4 +1,16 @@
 var request = require("request");
+
+var createInlineGif = function (selected) {
+    return '<img src="' + selected.images.original.url + '"/>';
+};
+var createRudeGifLink = function (selected) {
+    return '<a href="' + selected.images.original.url + '">Rude gif</a>';
+};
+var workSafeRatings = ["g", "y"];
```
Comment (meadsteve) [inline extensions/gif-me.js]: we could allow pg as well.

Comment (faroligo) [inline extensions/gif-me.js]: @meadsteve lets allow PG I reckon

Comment (faroligo) [inline extensions/gif-me.js]: Maybe even PG-13.  if its suitable for a 13 YO its proabably fine for work no?

**Comment by meadsteve** [comment]
@faroligo goodpoint. Now that I think about it we can always remove them later. Still SFWer than we have now.

**Pull request merged**